pyomo/contrib/fbbt/fbbt.py

```diff
@@ -52,11 +55,11 @@
 """
 
 
-class FBBTException(Exception):
+class FBBTException(PyomoException):
     pass
 
 
-def _prop_bnds_leaf_to_root_ProductExpression(node, bnds_dict):
+def _prop_bnds_leaf_to_root_ProductExpression(node, bnds_dict, feasibility_tol):
```

Comment:
@michaelbynum Can we document the keyword argument "feasibility_tol" or remove the documentation for the arguments?
This is true for most (all?) of the private methods that have this change.